tests/publish_public_data_json/test_sql/test/incremental_query_non_incremental_export_v1/metadata.yaml

```diff
@@ -0,0 +1,10 @@
+friendly_name: Test table for a non-incremental query
+description: >
+  Test table for a non-incremental query and incremental export
+owners:
+  - ascholtz@mozilla.com
+labels:
+  schedule: daily
+  public_json: true
+  incremental: true
+  incremental_export: false
```

Comment:
This is to test the behavior that is also expected from `ssl_ratios` and `deviations`: incremental query but non-incremental export